Graphics/OptionRowServiceChild Frame.lua

```diff
@@ -1,21 +1,29 @@
-local title_bg_width = _screen.w*WideScale(0.18,0.15)
+local row_bg_width = 358
+local row_bg_height = 30
+local row_bg_x = _screen.cx+92
+
+local title_bg_width = 115
+local title_bg_x = row_bg_x - row_bg_width/2
+
 local t = Def.ActorFrame{}
 t.InitCommand=function(self) self:x(WideScale(12,30)) end
 
 -- a row
 t[#t+1] = Def.Quad {
 	Name="RowBackgroundQuad",
-	InitCommand=function(self) self:zoomto(_screen.w * WideScale(0.475,0.54), _screen.h*0.0625):halign(0) end
+	InitCommand=function(self)
+	self:horizalign(center):x(row_bg_x):setsize(row_bg_width, row_bg_height)
+end
 }
 
 -- black quad behind the title
 t[#t+1] = Def.Quad {
 	Name="TitleBackgroundQuad",
 	OnCommand=function(self)
-		self:zoomto(title_bg_width, _screen.h*0.0625)
-			:halign(0):x(0)
+		self:setsize(title_bg_width, _screen.h*0.0625)
```

Comment:
for my education, why did you change from zoomto to setsize, and what's the difference between them?

Reply:
They do the same I don't know why I changed it lol